modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,77 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+:_content-type: PROCEDURE
+[id="proc-overriding-default-node-ip-selection-logic_{context}"]
+= Optional: Overriding the default node IP selection logic 
+
+You can create a hint file that includes the `NODEIP_HINT` variable to override the default IP selection logic. This allows users to select a specific node IP address from the subnet `NODEIP_HINT` variable and specify which IP address is used. For example, if a node has two interfaces, `eth0` and `eth1`, with the IP addresses `10.0.0.10/24` and `192.0.2.5/25` respectively, and the default route points to `eth0`, the node IP address normally uses the `10.0.0.10` address. With `NODEIP_HINT`, you can specify that the node IP address use `192.0.2.5` by configuring `NODEIP_HINT` to point to `192.0.2.1`. 
```

Comment:
@stevsmit @JoeAldinger 

So that's really kind of an CIDR range or subnet range. CIDRs are an esoteric network concept. I'm not familiar enough with the feature to suggest alternate text, though.

Specifically, these are subnet ranges:

> or example, a node that has two interfaces, `eth0` with an IP address `10.0.0.10/24`, and `eth1` with the IP address  `192.0.2.5/25`

Whereas _an IP address_ would refer to `1.2.3.4` or `192.168.1.1` or whatever.